### DISCUSSION TITLE
To Tinker or not to Tinker?

### DISCUSSION THREAD

**Discussion by joaomcteixeira** [THGLab/MCSCE#6] [Implementations]
@JerryJohnsonLee, because you have a running version of the code, could you investigate which features from Tinker does the code really uses so we can investigate the possibility to reimplement those features from scratch and thus avoid depending on Tinker?

**Comment by JerryJohnsonLee** [comment]
Sure I will look into that, but I may need to configure my environment a little bit more so that I can run Fortran programs directly on my local machine.

**Reply by joaomcteixeira** [reply]
Perfect. Keep in touch. If avoiding Tinker becomes hard. We can think about asking them the parts of their code that we need and we can create a stand-alone C/C++ code for that and bridge it with Python.

**Comment by joaomcteixeira** [comment]
today I was surfing again through the source code, and definitively what there is inside `source2.0303_v13.9/` is highly TINKER dependent. My original idea was to extract all parameters we needed from that code that but maybe we need to rephrase and rethink that. Going from inside out might not be a feasible strategy.

TINKER now has a [gitrepo](https://github.com/TinkerTools/tinker), and we can [download the linux read executables](https://dasher.wustl.edu/tinker/) (bottom of the page). I believe it will be easier in 2021 to use TINKER as a library that runs as external scripts if we decide so.

The tough thing here is to understand what from TINKER are we using. I will start recreating the steps of the program from your entry point.
